gen/com/intellij/plugins/haxe/lang/psi/HaxeAbstractClassDeclaration.java

```diff
@@ -31,8 +31,8 @@
   @Nullable
   HaxeComponentName getComponentName();
 
-  @Nullable
-  HaxeFunctionType getFunctionType();
+  @NotNull
```

Comment:
Anyway, we should pick one version of Grammar Kit and stick to it. I tried several versions from 1.1 to latest one and chose 1.2.0 because of fewest changes to project code.
Also 1.2.0 works with both IDEAs: 14 and 2016, and produces the same result.
Maybe I wrong and we should pick another version.